packages/workers-shared/router-worker/__tests__/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+	"compilerOptions": {
+		"target": "ES2020",
+		"module": "ES2020",
+		"lib": ["ES2020"],
+		"types": [
+			"@cloudflare/workers-types/experimental",
+			"@cloudflare/vitest-pool-workers"
+		],
```

Comment:
I think you need to add `"@types/node"` in here to fix TS checks.

Reply:
This did the trick, but actually we needed to change it in workers-shared. Still perplexed because I didn't touch any of the code in the root of workers-shared (the utils/helper.ts and script/* file already existed)

Reply:
seems like the ts checks failures are caused by the fact that `vitest.config.mts` was moved out of the root folder...whyyy would that break TS checks o.O????